### PR TITLE
Fix the menu | Logout | Adniya

### PR DESCRIPTION
**Pull request by adniyaYousaf**

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ const App = () => (
 			<Route path="/bidder-dashboard" element={<BidderDashboard />} />
 			<Route path="/list-tenders" element={<TendersList />} />
 			<Route path="/list-tenders/page/:pageNumber" element={<TendersList />} />
-			<Route path="/login" element={<LogInForm />} />
+			<Route path="/sign-in" element={<LogInForm />} />
```
Comment (soonick) [inline client/src/App.js]: Delete this route. Our sign-in form is already in `/`

File: client/src/Header.js [hunk 2]
```diff
@@ -7,23 +7,32 @@ import { post } from "./TenderClient";
...
-		setRole(userType);
+		const storageEventHandler = () => {
+			setRole(localStorage.getItem("userType"));
+		};
+
+		window.addEventListener("storage", storageEventHandler);
+
+		// return () => {
+		// 	window.removeEventListener("storage", storageEventHandler);
+		// };
```
Comment (soonick) [inline client/src/Header.js]: Uncomment

**Review by soonick — CHANGES_REQUESTED** [review]

File: client/src/App.js (hunk 1 above)
Comment (adniyaYousaf) [reply client/src/App.js]: Alright

File: client/src/Header.js (hunk 2 above)
Comment (adniyaYousaf) [reply client/src/Header.js]: Done

**Review by soonick — APPROVED** [review]

**Pull request merged**